maro/rl/training/trainer.py

```diff
@@ -60,12 +88,20 @@ def register_agent2policy(self, agent2policy: Dict[Any, str]) -> None:
     @abstractmethod
     def register_policy_creator(
         self,
-        global_policy_creator: Dict[str, Callable[[str], RLPolicy]]
+        global_policy_creator: Dict[str, Callable[[str], RLPolicy]],
     ) -> None:
+        """Register the policy creator. Only keep the creators of the policies that the current trainer need to train.
+
+        Args:
+            global_policy_creator (Dict[str, Callable[[str], RLPolicy]]): Dict that contains the creators for all
+                policies.
+        """
         raise NotImplementedError
 
     @abstractmethod
     def build(self) -> None:
+        """Build the trainer. This has to be called before the trainer can actually run.
```

Comment:
Create the required train-ops and replay memory. This should be called before invoking ``train`` or ``train_as_task``.